### ISSUE TITLE
How to compute loss on the test set

### ISSUE THREAD

**Issue by bouskaf** [fizyr/keras-retinanet#328]
Hello, thank you very much for this repository. I have trained a prediction model and everything seems to work fine. I can compute mean average precision (mAP) using evaluate.py on train/validation/test set. But I don't know how to compute loss (regression loss, classification loss) on validation set or test set respectively. Losses on train set are shown after each epoch during training.  My goal is to plot these like is shown [here](https://machinelearningmastery.com/display-deep-learning-model-training-history-in-keras/). But if I create history from _model.fit_generator_, it only contains  loss, regression loss, classification loss but no validation losses...  Thanks!

**Comment by GlastonburyC** [comment]
You need to specify a validation_data generator. Currently, the validation set is called using the eval.py method.

**Comment by bouskaf** [comment]
How simple is that... Thank you.
